exwm.el

```diff
@@ -1017,7 +1025,7 @@ FUNCTION is the function to be evaluated, ARGS are the arguments."
                          (car command-line-args) ;The executable file
                          "-d" (frame-parameter nil 'display)
                          "-Q"
-                         (concat "--daemon=" exwm--server-name)
+                         (concat "--fg-daemon=" exwm--server-name)
```

Comment:
This requires Emacs 26, but I think that's safe at this point.

Reply:
Maybe require Emacs 26 version here: https://github.com/ch11ng/exwm/blob/56db521a3ed6a2ccc1193110a56545a1992e9654/exwm.el#L8